ui/src/plugins/axios.ts

```diff
@@ -12,16 +14,37 @@ authAxios.interceptors.response.use(
     return Promise.resolve(resp);
   },
   error => {
+    if (error.response.status >= 400) {
+      /* display a snack bar with error, while enabling the user to submit a error report */
+      // TODO: Currently all failure reponses with 400+ status code will be displayed to the user.
```

Comment:
Keep the comment. We can revisit this if it becomes an issue. We need some practical experience with this to get more information.